cassandra/README.md

```diff
@@ -68,7 +68,7 @@ For keyspace `ycsb`, table `usertable`:
 * `cassandra.readconsistencylevel`
 * `cassandra.writeconsistencylevel`
 
-  * Default value is `ONE`
+  * Default value is `QUORUM`
```

Comment:
The note about durability in the face of node failure should get updated?

Reply:
removed line. Sorry, I did not see it before.